src/components/CertificatesList/messages.js

```diff
@@ -10,9 +10,9 @@ const messages = defineMessages({
   credentialsListEmpty: {
     id: 'credentials.list.empty',
     defaultMessage:
-      'No certificate available. Finish your first program to get a certificate.',
+      'No certificate available. Finish your first course or program to get a certificate.',
```

Comment:
There are no changes to renderEmpty in your PRI, so I'll write here.
I propose to add another version of the text when programs are not enabled on the platform (you can check if they are active in mfe config). If the programs are disabled, display the message “No certificate available. Finish your first course to get a certificate.”, and if they are enabled, display this.

Reply:
Platform doesn't have any flag (MFE and platform) indicating the programs availability.
If credentials service enabled then programs enabled too
MFE Config:
```
{
    "ACCESS_TOKEN_COOKIE_NAME": "edx-jwt-cookie-header-payload",
    "BASE_URL": "apps.local.openedx.io",
    "PUBLIC_PATH": "/learner-record/",
    "CREDENTIALS_BASE_URL": "http://credentials.local.openedx.io:8150",
    "CSRF_TOKEN_API_PATH": "/csrf/api/v1/token",
    "DISCOVERY_API_BASE_URL": "http://discovery.local.openedx.io:8381",
    "ECOMMERCE_BASE_URL": "http://localhost:18130",
    "ENVIRONMENT": "development",
    "LANGUAGE_PREFERENCE_COOKIE_NAME": "openedx-language-preference",
    "LEARNING_BASE_URL": "http://apps.local.openedx.io:2000",
    "LMS_BASE_URL": "http://local.openedx.io:8000",
    "LOGIN_URL": "http://local.openedx.io:8000/login",
    "LOGOUT_URL": "http://local.openedx.io:8000/logout",
    "STUDIO_BASE_URL": "http://studio.local.openedx.io:8001",
    "MARKETING_SITE_BASE_URL": "http://local.openedx.io:8000",
    "ORDER_HISTORY_URL": "http://localhost:1996/orders",
    "REFRESH_ACCESS_TOKEN_ENDPOINT": "http://local.openedx.io:8000/login_refresh",
    "SECURE_COOKIES": false,
    "SEGMENT_KEY": "",
    "SITE_NAME": "My Open edX",
    "USER_INFO_COOKIE_NAME": "user-info",
    "LOGO_URL": "http://local.openedx.io:8000/theming/asset/images/logo.png",
    "LOGO_TRADEMARK_URL": "http://local.openedx.io:8000/theming/asset/images/logo.png",
    "LOGO_WHITE_URL": "http://local.openedx.io:8000/theming/asset/images/logo.png",
    "FAVICON_URL": "http://local.openedx.io/favicon.ico",
    "MFE_CONFIG_API_URL": "/api/mfe_config/v1",
    "APP_ID": "learner-record",
    "SUPPORT_URL_LEARNER_RECORDS": "",
    "ENABLE_VERIFIABLE_CREDENTIALS": "true",
    "SUPPORT_URL_VERIFIABLE_CREDENTIALS": "",
    "INFO_EMAIL": "contact@local.edly.io",
    "PASSWORD_RESET_SUPPORT_LINK": "mailto:contact@local.edly.io",
    "DISABLE_ENTERPRISE_LOGIN": true,
    "COURSE_AUTHORING_MICROFRONTEND_URL": "http://apps.local.openedx.io:2001/authoring",
    "ENABLE_ASSETS_PAGE": "true",
    "ENABLE_HOME_PAGE_COURSE_API_V2": "true",
    "ENABLE_PROGRESS_GRAPH_SETTINGS": "true",
    "ENABLE_TAGGING_TAXONOMY_PAGES": "true",
    "INDIGO_ENABLE_DARK_TOGGLE": true,
    "AUTHN_MINIMAL_HEADER": false
}
```